libs/datasets/timeseries.py

```diff
@@ -1673,3 +1694,17 @@ def make_source_tags(ds_in: MultiRegionDataset) -> MultiRegionDataset:
     )
 
     return ds_in.replace_tag_df(pd.concat([source_df, other_tags_df]))
+
+
+def make_source_url_tags(ds_in: MultiRegionDataset) -> MultiRegionDataset:
+    """Make source_url tags from source tags"""
+    assert TagType.SOURCE_URL not in ds_in.tag.index.get_level_values(TagField.TYPE)
```

Comment:
I guess I don't quite understand the intent here.  Why do we need source_urls as their own tag type? shouldn't having `source` cover that case?

Reply:
I want to display count the URLs for the dashboard so need a way to get them out of all the `source` JSONs. This is just one way to make that happen.

Reply:
An alternative for https://github.com/covid-projections/covid-data-model/pull/958 is putting code that does roughly the same thing in the dashboard.

Reply:
It seems like that data would better live in the dashboard - or generally maybe some helper functions for accessing json data stored in the content (especially since I think all of the annotations store json now?)

Reply:
I added a TODO about cleaning this up when we know more about what we want.